packages/react-router/modules/index.js

```diff
@@ -6,4 +6,3 @@ export Router from './Router'
 export StaticRouter from './StaticRouter'
 export Switch from './Switch'
 export matchPath from './matchPath'
-export withRouter from './withRouter'
```

Comment:
Can we discuss what to do with `withRouter` in a separate PR? I think we may want to keep it, but just simplify the implementation.